cli/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -387,8 +387,10 @@ impl PathEnvironment {
             &System::from("x86_64-linux"),
         ];
         let manifest = if flox.catalog_client.is_some() {
+            tracing::debug!("creating raw catalog manifest");
```

Comment:
All of these tracing things are breadcrumbs that were missing while debugging why a `package.json`, etc wasn't detected during `init`.